index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portifolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+
+      <div class="toolbar">
+        <img class="nav-img" src="C:\Users\Ivan\Desktop\Setup-and-mobile-first\images\Toolbar - Dark - Compact.png">
+        <button class="btn"><i class="bars"></i></button>
+      </div>
```

Comment:
As requested by the previous reviewer, kindly change your `img` tags to background images.

Please notice that this image ("nav-img") shouldn't be an `img` element. Instead, it should be the [background image](https://www.w3schools.com/cssref/pr_background-image.php) of your container. This will make your code semantically accurate and give you an easier time with styling 😃. You can [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information.